extensions/test/NETCore.SetupTests/NETCore.SetupTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
💡 The first framework listed for `TargetFrameworks` controls the default view of a document opened inside Visual Studio. In the current form, documents open by default under a netcoreapp3.1 view. If you reverse them, they open by default under a net6.0 view. There are advantages and disadvantages to both approaches so I'll let you decide the one you want.

Reply:
Good callout, I had never noticed that.  I'd leave it up to the maintainers of this repo to main the determination on how they would like to have it organized.